### DISCUSSION TITLE
Unable to get VOUT working with a OUT1002_0 module

### DISCUSSION THREAD

**Discussion by 0x9c7d0d94** [artisan-roaster-scope/artisan#867] [Q&A] [answered]
HI there, hopefully this has an easy/quick answer as I am sure it is something I am missing.

What I have done:
- Added a OUT1002 module to the VINT hub:port number 1 (second port on hub)
- created an event slider (seen bellow) 
![Screenshot from 2022-04-11 19-15-41](https://user-images.githubusercontent.com/96889412/162803655-f8b5430f-7e9d-43ae-a6d2-6fb472aaf469.png)

What I expected to happen:
- voltage signal on OUT1002 when slider is moved

What I get:
- no signal

The module is detected when using the hello world script etc but it does not seem to talk to Artisan, with the other modules (inputs) I get a notice when unplugging them - with the OUT1002 I get no such notice.

It feels like I should be adding this as a new device, before the slider works... or is there another issue here?

**Comment by MAKOMO** [comment, marked as answer]
No need to define a device. You need to use the VOUT Command `out` set the voltage of a channel not `range`. Check the HELP page.

**Reply by 0x9c7d0d94** [reply]
Thank you, I had actually tried that at some point but the valve was not moving... I since took the module off and connected it to a multi meter and tried to measure the voltage but it did not register anything (at this point I had changed it to range and not out :( )

Thank you for the prompt reply, I can confirm it works and that I have a broken valve!
